tests/core/arch-x86/stalker-x86-exceptions.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2009-2021 Ole André Vadla Ravnås <oleavr@nowsecure.com>
+ * Copyright (C) 2010-2013 Karl Trygve Kalleberg <karltk@boblycat.org>
```

Comment:
As far as I can tell there's no code here that's derived from existing code, so these two lines seem confusing.